recipes/iscan/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "iscan" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f1b209154df73c67743e53b203fe41c354a079cdd943018a2f84aa642f27dd8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  requires:
+    - pytest <6
+  imports:
+    - iscan
+  commands:
+    - pytest --pyargs iscan
+    - iscan -h
```

Comment:
```suggestion
  requires:
    - pytest <6
    - pip
  imports:
    - iscan
  commands:
    - pytest --pyargs iscan
    - iscan -h
    - pip check
```

Reply:
Could you explain the addition of `pip check`? Given that dependency management and resolution is one of pip's weak points due to the lack of a full dependency -- one of the main reasons that prompted the creation of conda in the first place -- why is it necessary to add that here?

Reply:
``pip check`` is just one more layer of verification in case a new dependency is added to the ``setup.py`` it will verify if that dependency and version is at least already installed 
it is useful to catch some problems which may occur due to dependency change when updating the package version